config/routes.rb

```diff
@@ -12,6 +12,11 @@
     resources :users, except: :show
   end
 
+  get '/:city_uri', to: 'items#city'
+  get '/:city_uri/:category_uri', to: 'items#category'
+  get '/:city_uri/:category_uri/:subcategory_uri', to: 'items#subcategory'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.